proposals/lip-0071.md

```diff
@@ -28,20 +28,20 @@ In the current [Lisk protocol][lip-0042], the rewards are the same for all activ
+Currently, in the Lisk protocol if a validator misses a block, some other validators forge the block and the validator gets a default reward at current block height (`1 LSK` for mainchain). The reason is, the total rewards shouldn't exceed in a round it if a validator misses a block. Now, in the dynamic rewards, validators get rewards proportional to the validator weight. If a validator misses a block then a validator forging the block second time will get the reward proportional to their weight. This results in imbalancing the total reward in a round. In terms of inflation, currently in the Lisk mainchain, there is a maximum minting of `6*60*24*365 = 3153600` LSK per year. Every time a validator misses their block, 1 LSK less is minted per year. So the important aspect that we want to guarantee here is that in the dynamic rewards also never more than `3153600` LSK are minted per year so that there is a reliable upper bound on the inflation. In terms of incentives, it is also important that it should never be profitable for a validator if someone misses a block. Therefore, for missed blocks, validators forging blocks for the second time in the round, gets `getMinimumRewardActiveValidators(height)`. For the Lisk mainchain (assuming `101` active validators), the reward for missed blocks is 10<sup>7</sup> beddows or `0.1 LSK` at current block height. Since validators are already running full nodes and there is no additional effort for them to forge a missed block. There is a possibility that the minting of the tokens will be less if there are missed blocks.
```

Comment:
I find the first three sentences a bit confusing and also with some phrasal typos (e.g., "in a round it if a validator misses a block". Perhaps we could improve a bit here as well, since we touched the text anyways.

Reply:
Another example:

> 
If a validator misses a block then a validator forging the block second time will get the reward proportional to their weight. This results in imbalancing the total reward in a round.

If I understand well this does not actually happen. What we want to say here I guess is that "if we were granting dynamic block rewards even in the missed block case, then we would have this issue. But we don't right?

Reply:
Also, perhaps I am missing something, but by reading the text I don't understand why we give them 0.1 and not 1. Perhaps we should explain the reasoning. I don't remember actually why we decided to do so.

Reply:
Here it is the reasoning that if we give reward according to the weight for the missed blocks then it would imbalance the reward in a round.  
This line specifies this :  "If a validator misses a block then a validator forging the block second time will get the reward proportional to their weight. This results in imbalancing the total reward in a round." but actually we don't want this in the protocol and give minimum reward for the missed blocks.

Reply:
The reasoning why we give 0.1 LSK is, we don't want the total reward to exceed and validators are already running full nodes, there is no additional effort for them to forge a missed block.